pymc/model.py

```diff
@@ -1860,6 +1862,70 @@ def Deterministic(name, var, model=None, dims=None, auto=False):
     return var
 
 
+def _get_scaling(total_size, shape, ndim):
+    """
+    Gets scaling constant for logp
```

Comment:
I think these doc-strings are not correctly formatted. For example, they need new lines in a few places.